mne/viz/utils.py

```diff
@@ -67,6 +67,7 @@ def tight_layout(pad=1.2, h_pad=None, w_pad=None, fig=None):
 
     try:  # see https://github.com/matplotlib/matplotlib/issues/2654
         fig.canvas.draw()
+	fig.tight_layout(pad=pad, h_pad=h_pad, w_pad=w_pad)
         fig.set_tight_layout(dict(pad=pad, h_pad=h_pad, w_pad=w_pad))
```

Comment:
Fine with me :)

> On 02 Jun 2015, at 22:42, Eric Larson <notifications@github.com> wrote:
> 
> In mne/viz/utils.py:
> 
> > @@ -67,6 +67,7 @@ def tight_layout(pad=1.2, h_pad=None, w_pad=None, fig=None):
> >  
> >      try:  # see https://github.com/matplotlib/matplotlib/issues/2654
> >          fig.canvas.draw()
> > +	fig.tight_layout(pad=pad, h_pad=h_pad, w_pad=w_pad)
> >          fig.set_tight_layout(dict(pad=pad, h_pad=h_pad, w_pad=w_pad))
> Yes, that one line should fix things. My point is that this could be better organized as described above.
> 
> —
> Reply to this email directly or view it on GitHub.
>